sample-functions/SentimentAnalysis/handler.py

```diff
@@ -1,17 +1,31 @@
 import sys
+import json
 from textblob import TextBlob
 
+reload(sys)
```

Comment:
the setdefaultencoding method does not exist at this point in the program, I needed to reload the sys module for it to be available. Reason being I was seeing a ton of decoding errors when piping in tweets that had odd characters. Setting the default encoding to UTF-8 instead of ASCII solved those issues for me, although maybe there is a better way? I wouldn't think this would be necessary for all python functions, just use cases like this, where unexpected, odd characters can pop up.